custom_model_runner/datarobot_drum/drum/common.py

```diff
@@ -127,6 +117,78 @@ def validate_config_fields(model_config, *fields):
         )
 
 
+def validate_model_metadata_hyperparameter(hyper_params: Dict) -> None:
+    """Validate hyperparameters section in model metadata yaml.
+
+    Parameters
+    ----------
+    hyper_params:
+        Dictionray representative of hyperparameter section.
+
+    Raises
+    ------
+    DrumCommonException
+        Raised when validation fails.
+    """
+
+    def _validate_param_name(param_name: str):
+        allowed_char_re = r"[a-zA-Z]+[a-zA-Z_]*[a-zA-Z]+$"
+        if re.match(allowed_char_re, param_name) is None:
+            error_msg = (
+                "Invalid param name: {param_name}. "
+                "Only ENG characters and underscore are allowed. The parameter name should not start or"
+                " end with the underscore."
+            )
+            error_msg = error_msg.format(param_name=param_name)
+            raise DrumCommonException(error_msg)
+
+    def _validate_numeric_parameter(param: Dict):
+        param_name = param["name"]
+        param_type = param["type"]
+        min_val = param["min"]
+        max_val = param["max"]
+        default_val = param.get("default")
+        if min_val >= max_val:
+            error_msg = "Invalid {} parameter {}: min must be greater than max".format(
+                param_type, param_name
+            )
+            raise DrumCommonException(error_msg)
+        if default_val:
+            if default_val > max_val or default_val < min_val:
+                error_msg = "Invalid {} parameter {}: values must be between ({}, {})".format(
```

Comment:
Lets use brackets instead of parenthesis: http://www.differencebetween.info/difference-between-brackets-and-parentheses

Reply:
updated